docs/bring-your-own-certificates.md

```diff
@@ -0,0 +1,72 @@
+# Bring your own certificates
+
+The controller generates its own certificates when is deployed for the first time, it also manages the renewal for you.
+But you can also bring your own certificates so the controller can consume them as well.
+
+The controller consumes certificates contained in any secret labeled with `sealedsecrets.bitnami.com/sealed-secrets-key=active`,
+the secret has to live in the same namespace as the controller. There can be multiple such keys.
+
+Below you can find all the steps needed to create and consume your own certificates.
+
+## Set your vars
+```bash
+export PRIVATEKEY="mytls.key"
+export PUBLICKEY="mytls.crt"
+export NAMESPACE="sealed-secrets"
+export SECRETNAME="mycustomkeys"
+```
+
+## Generate a new RSA key pair (certificates):
+```bash
+openssl req -x509 -nodes -newkey rsa:4096 -keyout "$PRIVATEKEY" -out "$PUBLICKEY" -subj "/CN=sealed-secret/O=sealed-secret"
+```
+
+## Create a tls k8s secret, using your recently created RSA key pair:
+```bash
+kubectl -n "$NAMESPACE" create secret tls "$SECRETNAME" --cert="$PUBLICKEY" --key="$PRIVATEKEY"
+kubectl -n "$NAMESPACE" label secret "$SECRETNAME" sealedsecrets.bitnami.com/sealed-secrets-key=active
+```
+
+## Deleting the controller Pod is needed to pick they new keys:
+```bash
+kubectl -n  "$NAMESPACE" delete pod -l app.kubernetes.io/name=sealed-secrets
```

Comment:
```suggestion
kubectl -n  "$NAMESPACE" delete pod -l name=sealed-secrets-controller
```

(At some point we tried the app.kubernetes.io/name label but we had to roll that back due to upgrade regressions.
I'll try again to migrate to the standard labels once I find a way to ensure we don't break the upgrade path.)

Reply:
Sorry, but I dont get you...by default these are the [default Pod labels](https://github.com/helm/charts/blob/master/stable/sealed-secrets/templates/deployment.yaml#L25) you get, with the latest chart:

```
oc get pods --show-labels | grep seale
sealed-secrets-644b4c58bf-wf7kp                  1/1     Running     0          3h30m   app.kubernetes.io/instance=RELEASE-NAME,app.kubernetes.io/name=sealed-secrets,pod-template-hash=644b4c58bf
```

Also, `name=` is not a default label, it should be a [custom Pod label](https://github.com/helm/charts/blob/master/stable/sealed-secrets/values.yaml#L61)...

Reply:
Sorry, my bad, I get it now: https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.12.4/controller.yaml

The helm chart is not updated I would say...